### PR TITLE
Created folder for 2018-07-01 api version

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/readme.md [hunk 1]
```diff
@@ -745,7 +778,7 @@ directive:
   - suppress: DefinitionsPropertiesNamesCamelCase
     from: networkwatcher.json
     where: $.definitions.ProtocolConfiguration.properties.HTTPConfiguration
-    reason: Accidentally shipped with wrong casing – however fixing the casing is introducing a breaking change which is worse than living with the naming violation
+    reason: Accidentally shipped with wrong casing â€“ however fixing the casing is introducing a breaking change which is worse than living with the naming violation
```
Comment (lmazuel) [inline specification/network/resource-manager/readme.md]: Somthing wrong, please replace by a ASCII -

**Comment by vladbarosan** [comment]
@lmazuel nope. Tag @sarangan12 and @bsiegel since they are the ones who touched this file.

**Comment by lmazuel** [comment]
@bsiegel @sarangan12 seems you're the only two contributor on this script. Please help :)

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
Merging with broken linter to unlock Network team. Looking at the linter issue, this is pure in our utils, nothing to do with the PR.
I checkouted the PR on my machine and did:
- A manual `diff` to check that only API version changed.
- Check that VMSS files are still in their own api-version
That's enough for me to merge.

**Pull request merged**